src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -261,13 +261,15 @@ private void enableMenuBar(boolean b) {
 
 	private JMenu createSettingsMenu() {
 		JMenu menu = new JMenu(Translator.get("MenuSettings"));
+		menu.setMnemonic('S');
```

Comment:
I'm not a fan of single key mnemonics, same reason as using shift.  Change my mind!

Reply:
You may be mixing concept of mnemonic key with the accelerators ?
normally, mnemonics key are active only if in the menu, (F10 or "alt" pressed to active the tool bar) ( eg : "alt f" then "q" to quit ). 
(and also dont know how to make a not single key mnemonics ?  )